transport/transport.go

```diff
@@ -33,6 +35,9 @@ func NewHandler(us model.UserService) *Handler {
 		r.Delete("/", h.requireAdminUser(h.handleUser))
 	})
 
+	h.Mux.Patch("/organization/settings", h.requireAdminUser(h.updateSettings))
+	h.Mux.Get("/organization/settings", h.requireAdminUser(h.getSettings))
```

Comment:
These endpoints are available only to admin users. Authorization is performed by middleware in the transport layer. Or should the service layer also perform its own authorization?

Reply:
Not relevant anymore